s3-eventbridge-ecs/src/docker/README.md

```diff
@@ -0,0 +1,13 @@
+# Usage
+
+## Build the python task to read files from a S3 bucket when triggered by EventBridge
+
+* Build the Docker image first in local
+    
+    docker build -t `<image name>`:latest .
+
+* Publish the image to ECR and provide the image name, AWS region and AWS Account Number
+    
+    ./build.sh -i `<image name>` -r `<region>` -a `<account>`
```

Comment:
this file doesnt have permissions for executing. Add instructions on how to grant permissions or at least mention it.